src/App.js

```diff
@@ -1,42 +1,14 @@
-import "./App.css";
-import logo from "./logo.png";
+import { Footer, Navbar } from "components";
+import { Routes } from "routes/Routes";
+import "./styles/styles.css";
 
 function App() {
```

Comment:
You can change this as well to ES6 function expression

Reply:
yes, will change that to es6